src/tests/Common/Platform/platformdefines.h

```diff
@@ -177,8 +177,6 @@ error_t TP_itow_s(int num, LPWSTR buffer, size_t sizeInCharacters, int radix);
 error_t TP_itoa_s(int num, LPSTR buffer, size_t sizeInCharacters, int radix);
 LPWSTR TP_sstr(LPWSTR str, LPWSTR searchStr);
 LPSTR  HackyConvertToSTR(LPWSTR pwszInput);
-DWORD TP_CreateThread(THREAD_ID* tThread, LPTHREAD_START_ROUTINE worker,  LPVOID lpParameter);
```

Comment:
And the `THREAD_ID` typedefs in this file can be deleted too, I guess.

Reply:
There are many dead methods and types in this test, like `TP_SysAllocString`, `TP_SysStringLen` or `TP_DebugBreak`. A static analyzer can mark them all but I do not see enough value to do it now for this particular test.